Java/src/main/java/net/hypixel/api/util/RateLimiter.java

```diff
@@ -0,0 +1,62 @@
+package net.hypixel.api.util;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+public class RateLimiter {
```

Comment:
Should make sure to format this class similar to other classes in the project, using 4 spaces for indentation instead of tabs are the main thing